src/os/bluestore/BlueStore.cc

```diff
@@ -4830,7 +4830,7 @@ void BlueStore::Onode::rewrite_omap_key(const string& old, string *out)
   out->append(old.c_str() + out->length(), old.size() - out->length());
 }
 
-void BlueStore::Onode::decode_omap_key(const string& key, string *user_key)
+size_t BlueStore::Onode::get_userkey_offset_in_omap_key() const
```

Comment:
Can I please get "calc_" instead of "get_" ?
In BS we use this distinction when we retrieve a value that is not a property, but somehow a result of data structures / offsets.